From: Tommy Webb <tommy@calyxinstitute.org>
Date: Mon, 30 Oct 2023 18:22:40 +0000
Subject: Remove Legacy from adblocker setting title

It is the only ad-blocking option in CalyxOS Chromium, so it should not
be described as legacy.

Change-Id: If8f4353eb0994ada38940b56958e08ab7a7c7ec0
---
 chrome/browser/ui/android/strings/android_chrome_strings.grd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ui/android/strings/android_chrome_strings.grd b/chrome/browser/ui/android/strings/android_chrome_strings.grd
---  a/chrome/browser/ui/android/strings/android_chrome_strings.grd
+++ b/chrome/browser/ui/android/strings/android_chrome_strings.grd
@@ -220,7 +220,7 @@ CHAR_LIMIT guidelines:
 
       <!-- AdBlock settings -->
       <message name="IDS_PREFS_ADBLOCK" desc="Title of the AdBlock Settings screen. [CHAR-LIMIT=32]">
-        Legacy Adblock settings
+        AdBlock settings
       </message>
       <message name="IDS_OPTIONS_ADBLOCK_EDIT_TITLE" desc="The title of the screen that allows users to change the URL that is used to fetch the ad blocker filters.">
         Edit filters URL
-- 

